xml2rfc/util/unicode.py

```diff
@@ -34,6 +34,7 @@
     'street',
     'title',
     'u',
+    't',
```

Comment:
Please add, both here and below: dd, dt, li, blockquote, and any other block-level elements I missed.  Then add, both here and below, the "inline" elements: cref (?), em, eref (?), iref (?), relref, strong, sub, sup, tt, and xref.  (I didn't think much about the cross reference stuff, but I think they can contain text.)

I have immediate use for many of these.  I can wrap in `<t>` for some of the block-level elements, but not the inline-level ones.

Reply:
Note that with a list this long, you might consider how the latter list (bare_unicode_tags_with_notice) can be merged into the former (unicode_content_tags).

Reply:
I agree with the goal of the request. But I don't think we can do that without re-engaging the RSAB for a revised interpretation, or get a doc from RSWG through the RSAB that is explicit.

Reply:
Unfortunately, I have to agree with @rjsparks here. RFC 7997 allows non-ASCII in examples, but not e.g. in Mathematical formulæ or other cases in general text. Of course I agree with @martinthomson that it would be highly desirable to use non-ASCII for these cases, but I think that requires a change in policy.